src/java/org/apache/cassandra/gms/FailureDetector.java

```diff
@@ -62,14 +65,13 @@ public class FailureDetector implements IFailureDetector, FailureDetectorMBean
 
     private static long getMaxLocalPause()
     {
-        if (System.getProperty("cassandra.max_local_pause_in_ms") != null)
-        {
-            long pause = Long.parseLong(System.getProperty("cassandra.max_local_pause_in_ms"));
-            logger.warn("Overriding max local pause time to {}ms", pause);
-            return pause * 1000000L;
-        }
-        else
-            return DEFAULT_MAX_PAUSE;
+        long pause = MAX_LOCAL_PAUSE_IN_MS.getLong();
```

Comment:
`private static final long DEFAULT_MAX_PAUSE = 5000L * 1000000L; // 5 seconds` 

this has became unused property and might be removed